pkg/frontend/frontend.go

```diff
@@ -14,8 +14,10 @@ import (
 )
 
 var (
```

Comment:
looks like those are not just frontend errors and can appear everywhere, consider moving them to utils.go or jsonrpc.go

Reply:
moved to `jsonrpc.go`